wiki/Beatmap_Editor/Compose/en.md

```diff
@@ -1,406 +1,355 @@
-</tr>
-<tr class="even">
-<td><p>Distance Snap<br />
-(y)</p></td>
-<td><p><strong>Alt (Hold):</strong> Temporary toggle. Switch Beat Snap Divisor to Distance Snap<br />
-<strong>Alt + Mousewheel:</strong> Adjust Distance Snap multiplier.</p></td>
-<td><p>Distance between consecutive hit notes will be snapped based on their rhythmical difference. Best use while the timeline is paused.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Lock Notes<br />
-(l)</p></td>
-<td><p>(&quot;Select&quot; on a Hit note) <strong>Left-Click:</strong> Lock this note</p></td>
-<td><p>Lock selected note to current position and timestamp.</p></td>
-</tr>
-</tbody>
-</table>
-
-Bottom (Song's Timeline)
-------------------------
-
-<img src="BE_STL.jpg" title="fig:Song&#39;s Timeline" alt="Song&#39;s Timeline" width="200" /> At **bottom-left**, it shows **timestamp in milliseconds (ms)** and **song duration %**. The "%" may be switched to "intro/outro" when there is intro/outro storyboarding.
+### Centre-Right (Hit Sounds & Assist Tools)
+
+**List of available subtools complement to the original DIY toolkit.**
+
+![Hit Sounds & Assist Tools](BE_HSAT.jpg "Hit Sounds & Assist Tools")
+
+| Button (Keyboard Shortcut) | Usage | Description |
+| -------------------------- | ----- | ----------- |
+| New Combo (q) | **Right-click:** Change current note to next combo colour. | Note combo. This is equivalent to a verse / measure in song / instrument script. An acceptable range is about 5-20. |
+
+#### Hit Sounds
+
+| Button (Keyboard Shortcut) | Usage | Description |
+| -------------------------- | ----- | ----------- |
+| Whistle (w) | ("Select" on a Hit note) **Left-Click:** Use whistle sound on this note | Add a whistle sound addition to the selection. |
+| Finish (e) | ("Select" on a Hit note) **Left-Click:** Use finish sound on this note | Add a finish (cymbal) sound addition to the selection. |
+| Clap (r) | ("Select" on a Hit note) **Left-Click:** Use clap sound on this note | Add a clap sound addition to the selection. |
+
+#### Assist Tools
+
+| Button (Keyboard Shortcut) | Usage | Description |
+| -------------------------- | ----- | ----------- |
+| Grid Snap (t) | **Shift (Hold):** Temporary toggle. **Ctrl (Hold):** Temporary disable angle snapping. | Hit notes will be snapped to the grid by default. |
+| Distance Snap (y) | **Alt (Hold):** Temporary toggle. Switch Beat Snap Divisor to Distance Snap **Alt + Mousewheel:** Adjust Distance Snap multiplier. | Distance between consecutive hit notes will be snapped based on their rhythmical difference. Best use while the timeline is paused. |
+| Lock Notes (l) | ("Select" on a Hit note) **Left-Click:** Lock this note | Lock selected note to current position and timestamp. |
+
+### Bottom (Song's Timeline)
+
+![Song's Timeline](BE_HSAT.jpg "Song's Timeline")
+
+At **bottom-left**, it shows **timestamp in milliseconds (ms)** and **song duration %**. The "%" may be switched to "intro/outro" when there is intro/outro storyboarding.
```

Comment:
change to `At the **bottom-left**, there is the **timestamp** in milliseconds (ms) and **song duration** in percent. The percentage may be switched to "intro" or "outro" if there is storyboarding before or after the music.`